### PR TITLE
theme: added new theme Negligible

### PR DESCRIPTION
**Pull request by SagarYadav17**
### Prerequisites

- [ ] I have read and understand the `CONTRIBUTING` guide
- [ ] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

[Description of the change]

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/negligible.omp.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh3/main/themes/schema.json",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "segments": [
+                {
+                    "type": "os",
+                    "style": "powerline",
+                    "foreground": "cyan"
```
Comment (JanDeDobbeleer) [inline themes/negligible.omp.json]: It's prettier when this one has the `"prefix": ""` property. Not sure if this is on purpose, if so I'm inclined to merge, just my two cents as without that there's a space before the OS icon starts.

Comment (SagarYadav17) [reply themes/negligible.omp.json]: Thanks for mentioning that, I will fix that

**Comment by JanDeDobbeleer** [comment]
@SagarYadav17 can you squash those two commits into 1 and force push your branch? I'll merge right after that.

**Comment by SagarYadav17** [comment]
@JanDeDobbeleer sorry but I don't know how to do that. Any guidance.

**Comment by JanDeDobbeleer** [comment]
The easiest way, given no changes have been made to main on our end is to use the following commands:

```
git rebase -i main
```

That opens your git editor and you can mark the `fix:` commit as `fixup` and close the editor. This will merge that commit in the previous one and keep the first commits message. You can now force push your branch:

```
git push --force
```

Done 🙂

**Comment by SagarYadav17** [comment]
@JanDeDobbeleer Done 👍🏼

**Comment by JanDeDobbeleer** [comment]
Merged this manually.

**Pull request closed without merging**